mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala

```diff
@@ -86,7 +86,7 @@ object MLUtils {
         val indicesLength = indices.length
         while (i < indicesLength) {
           val current = indices(i)
-          require(current > previous, "indices should be one-based and in ascending order" )
+          require(current > previous, "indices should be one-based and in ascending order; found current=" + current + ", previous=" + previous + "; line=\"" + line + "\"")
```

Comment:
This line is too long (see spark style guide). Also, I recommend you use scala's string interpolation rather than manually building the string: 
```scala
s"previous=$previous, line='$line' "
```